### PR TITLE
sorter(ticdc): skip check sorter dir in unified sorter

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Skip check sorter dir because it has been checked when server starts.

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by liuzix — APPROVED** [review]

File: cdc/processor/pipeline/sorter.go [hunk 1]
```diff
@@ -89,11 +89,10 @@ func (n *sorterNode) Init(ctx pipeline.NodeContext) error {
 			log.Warn("File sorter is obsolete and replaced by unified sorter. Please revise your changefeed settings",
 				zap.String("changefeed-id", ctx.ChangefeedVars().ID), zap.String("table-name", n.tableName))
 		}
-		sortDir := ctx.ChangefeedVars().Info.SortDir
-		err := unified.CheckDir(sortDir)
-		if err != nil {
-			return errors.Trace(err)
-		}
+		// Sorter dir has been set and checked when server starts.
+		// See https://github.com/pingcap/ticdc/blob/674ac2/cdc/server.go#L280
```
Comment (amyangfei) [inline cdc/processor/pipeline/sorter.go:93]: `initDir` doesn't check the sort dir is writable

File: cdc/server.go [hunk 2]
```diff
@@ -257,27 +257,29 @@ func (s *Server) Close() {
...
 	if err != nil {
 		return errors.Trace(err)
 	}
-	diskInfo, err := util.GetDiskInfo(conf.DataDir)
+	log.Info(fmt.Sprintf("%s is set as data-dir (%dGB available), sort-dir=%s. "+
+		"It is recommended that the disk for data-dir at least have %dGB available space",
+		conf.DataDir, diskInfo.Avail, conf.Sorter.SortDir, dataDirThreshold))
+
+	// Ensure sorter dir exists and read-writable.
+	_, err = checkDir(conf.Sorter.SortDir)
```
Comment (amyangfei) [inline cdc/server.go:275]: This logic and `checkDir` function are not tested now

Comment (overvenus) [reply cdc/server.go:275]: This lint is tested in integration tests, without it, many tests fails.

`checkDir` is covered by unit tests now.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**